sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/ChangeFeedScan.scala

```diff
@@ -29,8 +29,10 @@ private case class ChangeFeedScan
     schema
   }
 
-  // TODO fabianm Implement
-  override def description(): String = super.description()
+  override def description(): String = {
+    // Ignore trailing $'s in the class names for Scala objects
+    this.getClass.getName.stripSuffix("$")
```

Comment:
the default is this already, if you don't implement:
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/Scan.java#L59-L61

Reply:
Fixed this with the new context that the description is used to create the physical query plan (for example returned by the explain command)